app/views/user/update_profile.html.slim

```diff
@@ -29,5 +29,28 @@
       tr
         th =f.label :location
         td =f.text_field :location, :placeholder => 'Where are you from?'
+
+    p.email_notifications Select which of the following notifications you want to receive:
+    table.form.email_notifications
```

Comment:
Where is the "Nightly emails on collections you work on" option?  (and let's list that first.)

Reply:
We don't have one of those, and we should - I think originally it was only page edits being sent overnight, and when we added additional information, I didn't think about it.